std/process.d

```diff
@@ -3233,11 +3242,15 @@ private:
     assertThrown(environment["std_process"]);
 
     // get() without default value
-    assert (environment.get("std_process") == null);
+    assert (environment.get("std_process") is null);
 
     // get() with default value
     assert (environment.get("std_process", "baz") == "baz");
 
+    // get() on an empty (but present) value
+    environment["std_process"] = "";
```

Comment:
I think you have triggered some sort of bug in the unittest code for windows with this.